app/src/main/res/layout/card.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.cardview.widget.CardView
```

Comment:
rename this file
card.xml is not good name

Reply:
inch dnem ba anun@